modules/src/tx_msg.rs

```diff
@@ -1,6 +1,7 @@
+use prost_types::Any;
 use tendermint::account::Id as AccountId;
 
-pub trait Msg: Sized {
+pub trait Msg: Clone {
```

Comment:
This `Clone` bound might cause issue if we ever need to deal with `Box<dyn Msg>` values as traits with `Clone` bounds cannot be made in trait objects. Not a problem at the moment, just pointing this out in case that comes up later on.